Project/Source/Components/UI/ComponentTransform2D.cpp

```diff
@@ -209,6 +209,10 @@ const float4x4 ComponentTransform2D::GetGlobalMatrixWithSize(bool view3DActive)
 	return globalMatrix * float4x4::Scale(size.x, size.y, 0);
 }
 
+void ComponentTransform2D::SetDirty(bool value) {
+	dirty = value;
+}
+
```

Comment:
A public variable would suffice in this case, but no need to change (Y)

Reply:
This function is a duplicate. Invalidate has the same functionality.

Reply:
@JaimeBea So you want me to just call InvalidateHierarchy? My intention was to have the same call "logic" in the ModuleUserInterface, so everything gets sorted out in the next frame. You tell me if you want me to delete it.

Reply:
Yeah, you should call Invalidate or InvalidateHierarchy so that everything gets updated.

Reply:
I think there are other components with SetDirty that should do the same.

Reply:
Cleared